packages/scene-graph-mediator/client/src/modules/parseArgs.ts

```diff
@@ -171,5 +175,12 @@ export default function parseArgs(): Args {
 
   args.assetRoot = args.assetRoot.replace(/\/$/, '');
 
+  for (let i = 0; i < args.plugins.length; i++) {
+    const plugin = args.plugins[i];
+    if (isRelativePath(plugin)) {
```

Comment:
`path.isAbsolute` is more handy.

```
> path.isAbsolute('/aaaa/bbbb/cccc')
true
> path.isAbsolute('/aaaa/../cccc')
true
> path.isAbsolute('./aaaa/cccc')
false
```

Reply:
In this case, Is it make mistakes?

```
> path.isAbsolute('ModuleName')
false
```

It isn't relative path.
!absolute !== relative.

Reply:
So there is another path variant.

- absolute
- relative
- resolved

I'd like to forget about `$NODE_PATH` because it can be defined by user or user's environment, will you be supporting for this variable ?

Reply:
It basically worked correctly.
https://github.com/drecom/scene-graph/blob/de696ada51a1b614192b41134ecc817c619559f1/packages/scene-graph-mediator/client/src/exporter/ExportManager.ts#L70

But, The following points need to be corrected.
Plugins have been removed below.

https://github.com/drecom/scene-graph/blob/de696ada51a1b614192b41134ecc817c619559f1/packages/scene-graph-mediator/client/src/modules/parseArgs.ts#L19

Reply:
Added support for loading plugins by specifying "ModuleName".
74bb88c 3dae03e